Pizzeria/Pizzeria/Employee/Employee.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Pizzeria
+{
+    public class Employee
+    {
+        public decimal Salary { get; set; }
+        public string Name { get; set; }
+        public int Age { get; set; }
+        public int WorkExperience { get; set; }
```

Comment:
At lease hide the setters, otherwise the state of the object can be changed from any place in the code